maps/005-maze.txt

```diff
@@ -1,9 +1,9 @@
 ###           #                      #H  |
 #P#           #                      ### |
 # #B          #                 #    #   |
-# ##  #  #BB# #              #  #    # ##|
+# ##  #  #BB  #              #  #    # ##|
```

Comment:
The red key is... *dons sunglasses*... a red herring.

There is an obscure method with which you can escape with the red key, in order to discover that there's no level progression in it. You have to hold up and jump on top of the key to get out. Either way, I'd like to keep this as a trap.

The level is completable without the red key.

Reply:
Ah ok, it *is* that, I did wonder. I've put the block back :)